src/common/errorHandler/errorHandler.component.js

```diff
@@ -0,0 +1,74 @@
+import Element from '../../elements/element/element.component';
+import {helper} from '../helper'
+import './errorHandler.style.less';
+
+let errorHandlerSingleton;
+let bindClickEvent;
+
+export class ErrorHandler extends Element {
+  constructor(error) {
+    if (errorHandlerSingleton) {
+      return errorHandlerSingleton
+    }
+
+
+    errorHandlerSingleton = super('section');
```

Comment:
It's quite difficult to understand result tree of elements and change something if you need. Think how to make it better.